pyproject.toml

```diff
@@ -34,7 +34,7 @@ dependencies = [
     "ipywidgets",
     "pydantic>=2.0",
     "requests",
-    "qiskit>=1.0, <1.2",
+    "qiskit>=1.2, <1.3",
```

Comment:
Should we set the maximum Qiskit version to `1.2.4` ? (The one the patch of the function `substitute_node_with_dag` is based one)
To be 100% sure any new version Qiskit `1.2.x` doesn't break our patch by changing the overwritten function ?
But we will have create a new release for each new patch of `1.2` then.

Reply:
In that case it's not even a question of maximum version but we would need a fixed version. The file might not be the same already between 1.2.0 and 1.2.4.

Reply:
Yes true

Reply:
Then let's fix the version that way we'll avoid compatibility issues and/or regressions.